xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd

```diff
@@ -414,7 +414,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="positionAtBusStop"/>
 			<xsd:enumeration value="boatGangway"/>
 			<xsd:enumeration value="ferryGangway"/>
-			<xsd:enumeration value="telecabineplatform"/>
```

Comment:
Is there any reason why the casing is not the same between the two instances? If we change this, can't we directly align it?

Reply:
done